srtcore/channel.cpp

```diff
@@ -68,6 +71,9 @@ modified by
    #include <ws2tcpip.h>
    #include <mswsock.h>
 #endif
+*/
```

Comment:
I commented out, then can be removed?

Reply:
I think so, yes. I left it just for the case when it might cause problems on other platforms and the contents of `platform_sys.h` file should be fixed. But I think there's nothing to worry about.